src/material-examples/example-module.ts

```diff
@@ -601,22 +601,22 @@ export const EXAMPLE_COMPONENTS: {[key: string]: LiveExample} = {
     title: 'Basic toolbar',
     component: ToolbarOverviewExample
   },
-  'tooltip-overview': {
-    title: 'Basic tooltip',
-    component: TooltipOverviewExample
-  },
   'tooltip-delay': {
-    title: 'Tooltips with delay',
+    title: 'Tooltip with a show and hide delay',
```

Comment:
This file is auto-generated. You need to update the `@title` in each example.

Reply:
Sorry (should have mentioned it in the PR description): those are the up-to-date descriptions from the actual `@title` comments. The titles recently got updated, but the `example-module` just didn't get updated.